s3_file_field/views.py

```diff
@@ -106,20 +163,24 @@ def upload_finalize(request: Request) -> HttpResponseBase:
     # ):
     #     raise BadSignature()
 
-    _multipart.MultipartManager.from_storage(field.storage).finalize_upload(finalization)
+    try:
+        size = _multipart.MultipartManager.from_storage(field.storage).get_upload_size(object_key)
```

Comment:
Add a code comment stating that this has a side effect of ensuring the object exists, and that is critical to this logic.